server/lib/enrich/article.js

```diff
@@ -39,7 +39,10 @@ module.exports = exports = function article(content, contract, graphicSyndicatio
 			)
 
 	content.canAllGraphicsBeSyndicated =
-		graphicEmbeds &&
+		// From MDN docs: every acts like the "for all" quantifier in mathematics.
+		// In particular, for an empty array, it returns true.
+		// So if the filter above returned an empty array, [].every(whatever) would return `true`
+		graphicEmbeds.length > 0 &&
 		graphicEmbeds
 			.every((item) => item.canBeSyndicated === 'yes');
```

Comment:
Good spot! 👍 

Should we first check that `graphicEmbeds` is not false as well i.e,

```js
graphicEmbeds &&
graphicEmbeds.length > 0 &&
graphicEmbeds.every((item) => item.canBeSyndicated === 'yes');
```

Given that `graphicEmbeds` could be returned as `false` if embeds array isn't present in content.

Reply:
Fair point!